interface/forms/eye_mag/js/eye_base.php

```diff
@@ -2178,7 +2178,8 @@ function showpnotes(docid) {
 function getTimeStamp() {
     var now = new Date();
     var AMPM = now.getHours() >= 12 ? 'PM' : 'AM';
-    return now.getHours() + ':' + ((now.getMinutes() < 10) ? ("0" + now.getMinutes()) : (now.getMinutes())) + AMPM;
+    var hours = ( (now.getHours() > 12) ? ( now.getHours() - "12") : (now.getHours()) );
+    return hours + ':' + ((now.getMinutes() < 10) ? ("0" + now.getMinutes()) : (now.getMinutes())) + AMPM;
```

Comment:
hi @ophthal , do you prefer this change or the change [here](https://github.com/openemr/openemr/pull/8101/files)?

Reply:
The second one is easier to read.